### ISSUE TITLE
webidl: Why remove the underline of a variable or function

### ISSUE THREAD

**Issue by ucanudo** [emscripten-core/emscripten#12495]
![image](https://user-images.githubusercontent.com/45194746/95725477-4a922f00-0caa-11eb-8393-e6d6b80a3bdf.png)

`
class Test{
public:
 Test();
~Test();
void _getInnerData(){
 return _innerData;
}
double _innerData;
}
`
webidl will remov '_' . convert '_getInnerData()'  to 'getInnerData()', convert '_innerData' to 'innerData'. 
why???

**Comment by kripken** [comment]
I think that's in the WebIDL parser code that we didn't write (which is 99% of it). I'm not familiar enough with WebIDL to know if prefixes like that matter, but that code was taken from Mozilla's codebase, so it's probably right.

**Comment by ucanudo** [comment]
> I think that's in the WebIDL parser code that we didn't write (which is 99% of it). I'm not familiar enough with WebIDL to know if prefixes like that matter, but that code was taken from Mozilla's codebase, so it's probably right.

Thank you very much.  What's the difference between webidl in Emscripten and this site (https://heycam.github.io/webidl/),   Why are some keywords not recognized? Does the file 'webidl.py' is too old?

**Comment by kripken** [comment]
Yes, it's probably just that we took that code many years ago, and have not updated. In practice just a subset of WebIDL matters, which is why it wasn't urgent to update. Is something missing?
